Sources/Danger/GitHubDSL.swift

```diff
@@ -138,6 +139,9 @@ public extension GitHub {
 
         /// Possible link relations
         public let links: Link
+
+        /// The labels associated with this pull request.
+        public let labels: [Label]
```

Comment:
Are the labels actually coming here in the JSON?
Every pull request is also an issue (this is how GitHub handles PRs I think), so `github.issue` should be present for every pull request, so you can get the labels from there I support

Reply:
Ah, you’re right. I read that every PR is also an issue but somehow haven’t thought about using `github.issue` in my Dangerfile. Thanks for the hint and your quick answers, I’ll close this PR in this case. Wouldn’t be a good idea to deviate from the GitHub API here, I think.

Reply:
But to answer your question: Yes, the labels are actually coming in the JSON. Even the JSON for the unit test that I updated in this PR has one.